ChopChop/Models/User/CurrentUser.swift

```diff
@@ -0,0 +1,3 @@
+import Foundation
+
+var USER_ID = UserDefaults.standard.string(forKey: "userId")
```

Comment:
Is it better to store this in the UserSetting Object and use it as environmentObject that is accessible to all subviews. So like when the app starts, the user setting will get the id from UserDefaults and we just use the environment object from then on. Not sure if that is how environmentObject works.